percona-packaging/scripts/psmdb_builder.sh

```diff
@@ -201,6 +201,19 @@ install_golang() {
     ln -s /usr/local/go1.11 /usr/local/go
 }
 
+install_python_packages_by_pip() {
+    get_system
+
+    if [ "x${RHEL}" == "x6" ]; then
+        pip2.7 install --user -r buildscripts/requirements.txt
+    elif [ "x${RHEL}" == "x8" ]; then
+        /usr/bin/pip3.6 install --user typing pyyaml regex Cheetah3
```

Comment:
as we are reworking script please check the possibility to use buildscripts/requirements.txt for rhel8